apps/cookbook/src/app/examples/slides-example/slides-example.component.html

```diff
@@ -0,0 +1,28 @@
+<kirby-slides
+  [slidesOptions]="slidesOptions"
+  [slides]="slides"
+  (selectedRecord)="getDataFromActiveSlide($event)"
```

Comment:
Is this the selected slide? if so please rename to avoid confusion of what it is

Reply:
Har tilføjet en comment